README

```diff
@@ -34,6 +34,56 @@ xtensa-byt-elf-objcopy -I binary -O elf32-xtensa-le -B xtensa image.bin image.bi
 
 xtensa-byt-elf-objdump -D image.bin.elf > image.dis.txt
 
+
+logger
+======
+
+logger is used to print logs delivered from FW dma_trace mechanism, by searching log
+entries in *.ldc file generated by rimage
+
+e.g. gets traces from "/sys/kernel/debug/sof/etrace" file and prints logs to 
+     stdout
+
+logger -l ldc_file
+
```

Comment:
I have a basic question, is there a default ldc_file?
I wonder if the ldc_file could be generated when we build the firmware.  And if there is a default ldc file, can we save this parameter  sometimes?

Reply:
ldc_file is generated always when we build the firmware and its name differs for particular platforms (sof-apl.ldc, sof-cnl.ldc etc.) - also *.ldc file content could be different for platforms since they can have different log_messages. For that reason, in my opinion it is hard to define deafult *.ldc file

Reply:
@mengdonglin Please refer to a high level documentation of the process being delivered in https://github.com/thesofproject/sof-docs/pull/51.